### PR TITLE
support symbol resolution of short-lived process. 

### PR DESCRIPTION
**Pull request by vijunag**
Support symbol resolution for short-lived processes (#2014)
New command line options have been added to tools/trace.py to support the new BUILD_ID stackmap. List of symbol files can be added to the script to resolve symbols from build id as reported by the kernel in the stack trace. A comma separated list of symbol files can be passed to the script as shown below. 

Eg invocation to trace inet_pton(as mentioned in #2014)
python tools/trace.py -s /lib/x86_64-linux-gnu/libc.so.6,/bin/ping  'p:c:inet_pton' -U

### CODE REVIEW

**Comment by yonghong-song** [comment]
@brendangregg Now bcc core has stackmap buildid support. This patch tries to amend `trace.py` for short lived processes. What do you think for `trace.py`? What else in bcc we may utilize buildid support? `profile.py`?

**Comment by vijunag** [comment]
> Since a new flag is introduced, the example file and man page needs changes as well.

Updated the man file.

**Comment by yonghong-song** [comment]
[buildbot, test this please]

File: tools/trace.py [hunk 1]
```diff
@@ -693,6 +697,8 @@ def __init__(self):
                   help="print CPU id")
                 parser.add_argument("-B", "--bin_cmp", action="store_true",
                   help="allow to use STRCMP with binary values")
+                parser.add_argument('-s', "--sym_file_list", type=str, metavar="SYM_FILE_LIST",
```
Comment (yonghong-song) [inline tools/trace.py]: Maybe break the line so that it will remain  maximum 80 characters per line?

File: tools/trace.py [hunk 2]
```diff
@@ -693,6 +697,8 @@ def __init__(self):
...
+                  dest="sym_file_list", help="coma separated list of symbol files to use for symbol resolution")
```
Comment (yonghong-song) [inline tools/trace.py]: break into two lines?

**Comment by vijunag** [comment]
Please note that test bed failures are not due to this change.

**Comment by yonghong-song** [comment]
[buildbot, test this please]

**Comment by yonghong-song** [comment]
Thanks. The change looks good to me.

**Pull request merged**